src/dialog/youmu.c

```diff
@@ -830,18 +849,22 @@ DIALOG_TASK(youmu, StageExPostBoss) {
 
 	FACE(youmu, smug);
 	MSG(youmu, "Precisely.");
+
 	FACE(elly, blush);
 	MSG(elly, "Yōmu, you’re absolutely incredible!");
+
 	FACE(youmu, chuuni);
 	MSG(youmu, "Flattery will get you everywhere, my dear.");
 	MSG(yumemi, "After all that effort you put into stealing the Tower from me…");
+
+	MSG(yumemi, "… that’s all it took for you to give up on your vision, Elly? A delusional katana-wielding woman with cheesy action-hero one-liners?");
```

Comment:
I brought this up before, and I still think this line would work better/be funnier with some more disparaging language

```suggestion
	MSG(yumemi, "… that’s all it took for you to give up on your vision, Elly? A deranged sword girl with a bunch of cheesy action-hero one-liners?");
```


But it's up to you; I don't feel strongly about it.